### PR TITLE
Switch to yarn

### PR DESCRIPTION
**Pull request by wwwdata**
There's currently one interesting discussion ongoing: https://github.com/yarnpkg/yarn/issues/570

the behaviour at the moment is that when a normal `yarn install` is running, it automatically generates and updates the `yarn.lock` file within the specified version range from the `package.json`

So in order to reproduce the correct dependencies always purely from the lock file it's important to add `--pure-lockfile` to the `yarn install` command ... I hope they change that. But just in case people are wondering here why the hell the yarn.lock changes after an install.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "eslint": "3.7.1",
```
Comment (wwwdata) [inline package.json]: @fspoettel don't your just thumbs up, give me a real review and approve!!! 😄

Comment (fspoettel) [inline package.json]: i thought i did, looks like i did not push the Approve button hard enough

Comment (wwwdata) [inline package.json]: it's ok, just make sure you are fully concentrated next time. It's important to stay focused when pressing buttons!

Comment (fspoettel) [inline package.json]: giphy

**Pull request merged**